README.md

```diff
@@ -16,45 +80,114 @@ Generating this manifests is very easy, you just need add the proper dependency
  </dependency>
 ```
 
-We also need to add an annotation to enable dekorate. In this case we will use `@KubernetesApplication` which also gives us access to more Kubernetes specific configuration options.
-Edit the App class and add the indicated annotation.
+We need to enable `Dekorate` by adding an annotation. In this case we will use `@KubernetesApplication` which also gives us access to more Kubernetes specific configuration options.
+Edit the App class and add the following annotation.
 
 ```
-@KubernetesApplication(name = "hello-world-fwless-k8s", ports = @Port(name = "web", containerPort = 8080))
+@KubernetesApplication(
+        name = "hello-world-fwless-k8s",        
+        ports = @Port(name = "web", containerPort = 8080),  
+        host = "fw-app.127.0.0.1.nip.io", //2
+        expose = true, //3
+        imagePullPolicy = ImagePullPolicy.Always //4
+)
 ```
-We need to specify a port in order to prevent dekorate it is about a Web Application, this way a service will be generated in the k8s manifest.
+- We need to specify a **`@Port`** in order to prevent dekorate it is about a Web Application, this way a service will be generated in the k8s manifest.
```

Comment:
This sentence is not correct in english and confusing. I suppose that you would like to say that we want to specify within the pod definition that the web application is running using the port : 8080 and this port will also be used to expose a service using the same value ...

Reply:
The Port tells to Dekorate that a service should be created. I will try to explain clearer